### PR TITLE
Add text normalization

### PR DESCRIPTION
**Pull request by sorenmulli**
I think the warning thing might be über-careful - shouldn't I just remove that?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
 # Punctuation restoration 
-Adds punctuation and capitalization for a given text.
+Adds punctuation and capitalization for a given text without punctuation.
 
 Works on Danish, German and English. 
 
-Models hosted on huggingface! <3 
+Models hosted on huggingface! ❤️  🤗
```
Comment (kshll6) [inline README.md:6]: nice

File: punctfix/inference.py [hunk 2]
```diff
@@ -37,22 +44,30 @@ class PunctFixer:
     """
     PunctFixer used to punctuate a given text.
     """
+    word_normalization_pattern = re.compile(r"[\W_]+")
```
Comment (kshll6) [inline punctfix/inference.py:47]: Maybe just move out of class as constant

**Review by kshll6 — CHANGES_REQUESTED** [review]

**Comment by sorenmulli** [comment]
Approved, @kshll6?

**Comment by kshll6** [comment]
> approved!

**Review by kshll6 — APPROVED** [review]

**Pull request merged**